hooks/command

```diff
@@ -187,14 +187,14 @@ else
   do
     set +e
     pod_json=$(kubectl get pod "$pod_name" -o json)
+    set -e
     init_container_status="$(echo "$pod_json" | jq ".status.initContainerStatuses[0].state.terminated.exitCode")"
     if [[ -n "$init_container_status" && "$init_container_status" != "0" ]]; then
       echo "Warning: init container failed with exit code $init_container_status, this usually indicates plugin misconfiguration or infrastructure failure"
       status="$init_container_status"
     else
-      status="$(echo "$pod_json" | jq ".status.containerStatuses[0].state.terminated.exitCode")"
+      status="$(echo "$pod_json" | jq -e ".status.containerStatuses[0].state.terminated.exitCode")"
```

Comment:
`status` can end up being null atm, we can instead check the jq exit code first:
```sh
container_status="$(echo "$pod_json" | jq -e ".status.containerStatuses[0].state.terminated.exitCode")"
if "$?" -eq 0; then
  status=container_status
end
```